lib/protobuf/nats/server.rb

```diff
@@ -22,6 +22,7 @@ def initialize(options)
         @thread_pool = ::Protobuf::Nats::ThreadPool.new(@options[:threads], :max_queue => max_queue_size)
 
         @subscriptions = []
+        @server = options.fetch(:server, first_ipv4_address)
```

Comment:
I'm thinking I should change the default to `Socket.gethostname`. I wanted the ipv4 address to mirror what we have today, but I actually do a reverse lookup on the ipv4 address to in logstash to get the hostname anyways. That lookup has been crashing the QA logstash recently due to timeouts. What say ye?